config/csi-rbac/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+# Adds namespace to all resources.
+namespace: csi-system
```

Comment:
default ns must match the operator default ns, if not it will be unusable when generating without customization

Reply:
This is following the standard what kustomization says, the namespace should have the name suffix